packages/core/bin/gen-version.js

```diff
@@ -1,5 +1,5 @@
 #!node
-import { writeFileSync, readFileSync } from 'fs';
+const { writeFileSync, readFileSync } = require('fs');
```

Comment:
I think we can skip using `gen-version` and just rely on `npm_package_version` env variable?

Reply:
If a package is being bundle though, do we get the env variable of the agent or the package?

Reply:
LE: just tested. `npm_package_version` is unreliable as it is not actually replaced (tried tsc/ts-node) but left there in the compiled code. Guess what happens when you load a library that relies on `npm_package_version` internally... it is actually getting the version from the `package.json` from where you run the `npm start cmd` or `yarn cmd` :-/